src/hooks/useCountdownTimer.ts

```diff
@@ -51,7 +51,7 @@ export const useFormattedTime = (durationInMinutes: number) => {
       setFormattedTime(formatted)
     },
     onFinish: () => {
-      // Дополнительные действия по завершению таймера, если необходимо
+      // Additional actions to end the timer, if necessary
     },
```

Comment:
I see commits from https://github.com/Sunagatov/Iced-Latte-Frontend/pull/82 there, it makes review process difficult. I kindly ask you to make PR isolated in the future 🙏  FE review is hard enough itself, let alone dealing with repeating commits 😄